src/main/resources/application.properties

```diff
@@ -11,9 +11,9 @@ quarkus.cache.caffeine."glob-cache".maximum-size=200
 hibernate-github-bot.jenkins.github-app-id=347853
 hibernate-github-bot.develocity.uri=https://ge.hibernate.org/
 
-quarkus.openapi-generator.codegen.spec.gradle_enterprise_2023_4_api_yaml.config-key=develocity
+quarkus.openapi-generator.codegen.spec.gradle_enterprise_2023_4_api_yaml.config-key=gradle_enterprise_2023_4_api_yaml
```

Comment:
The whole point of this line was to us a simpler key 🙂 I take it it no longer works?

Reply:
yeah 😖 I've opened this PR https://github.com/quarkiverse/quarkus-openapi-generator/pull/848 it should allow to use keys again